t/gabeacon.t

```diff
@@ -497,6 +497,25 @@ my $expected_data8 = {
   "datasetAlleleResponses" => undef
 };
 
+my $allele_request_2 = {
```

Comment:
I see you do not test your URL builder above.

Reply:
Which one? The one above doesn't have a dataset response and without a dataset, there's no url.

Reply:
Ideally you would mock up some test data that would work that part of the code and show the effect of none/one/many URLs. I appreciate that it's a royal pain.